### DISCUSSION TITLE
Installation Issues

### DISCUSSION THREAD

**Discussion by MademoiselleVictoire** [cmus/cmus#1148] [Troubleshooting]
I'm trying to install cmus on Oracle Linux 8 UEK, but each time I run ./configure it fails. I had no issue in the past installing on Debian any version.

**Comment by nefthy** [comment]
Please post the output of `./configure`

**Comment by MademoiselleVictoire** [comment]
> Please post the output of `./configure`

Here's the output... Am I missing any additional packages..?

![20211220_194547.jpg](https://user-images.githubusercontent.com/53003138/146852040-75242330-c019-43c5-ad2f-0004f65a5b37.jpg)

**Comment by nefthy** [comment]
you are missing the ncurses library and headers.

**Reply by MademoiselleVictoire** [reply]
Thanks! sudo dnf install -y ncurses-devel resolved the dependencies.
